ioservice/cob_foms.c

```diff
@@ -1225,6 +1243,37 @@ static int ce_stob_edit(struct m0_fom *fom, struct m0_fom_cob_op *cd,
 	return M0_RC(rc);
 }
 
+static int cob_bytecount_decrement(struct m0_cob *cob, struct m0_cob_bckey *key,
```

Comment:
I am not sure if this is a coding style requirement. There are other existing functions written in a similar manner eg:
```
static void cob_op_credit(struct m0_fom *fom, enum m0_cob_op opcode,
                          struct m0_be_tx_credit *accum);
```
```
static int ce_stob_edit_credit(struct m0_fom *fom, struct m0_fom_cob_op *cc,
                               struct m0_be_tx_credit *accum, uint32_t cot);
```
etc.

Reply:
Yes, there is existing code that is written variously. It does not mean we should continue such with fresh code.
Also, while it may not be mentioned in the Coding guidelines document, one can surely expect a uniform style of writing code within a single PR.